picky-signtool/src/verify.rs

```diff
@@ -244,14 +249,29 @@ fn apply_flags<'a>(
     }
 }
 
-pub fn extract_signed_ps_file_content(raw_content: String) -> String {
-    let end = match raw_content.find(PS_AUTHENTICODE_HEADER) {
-        Some(index) => index - 2, // -2 to remove \r\n from the `# SIG # Begin signature block` line
+pub fn extract_signed_ps_file_content_bytes(raw_content: &[u8]) -> &[u8] {
+    // raw_content - utf-16 encoded bytes
+    let find = |haystack: &[u8], needle: &Vec<u8>| -> Option<usize> {
```

Comment:
Taking `&[u8]` is always better than `&Vec<u8>` when possible
```suggestion
    let find = |haystack: &[u8], needle: &[u8]| -> Option<usize> {
```